src/parser/transform/statement/transform_create_table.cpp

```diff
@@ -95,6 +96,31 @@ unique_ptr<CreateStatement> Transformer::TransformCreateTable(duckdb_libpgquery:
 		case duckdb_libpgquery::T_PGColumnDef: {
 			auto cdef = (duckdb_libpgquery::PGColumnDef *)c->data.ptr_value;
 			auto centry = TransformColumnDefinition(cdef);
+			if (centry.type.IsSerial()) {
+				// create a sequence associated with the column
+				auto sequence = make_unique<CreateSequenceInfo>();
+				string seq_name = info->table + "_" + centry.name + "_seq";
```

Comment:
postgres-style naming for the sequence associated with the serial column

Reply:
LGTM